src/libsystemd/sd-journal/journal-file.c

```diff
@@ -512,6 +520,8 @@ static bool warn_wrong_flags(const JournalFile *f, bool compatible) {
                                         strv[n++] = "zstd-compressed";
                                 if (flags & HEADER_INCOMPATIBLE_KEYED_HASH)
                                         strv[n++] = "keyed-hash";
+                                if (flags & HEADER_INCOMPATIBLE_COMPACT)
```

Comment:
FLAGS_SET?

Reply:
I think we only recommend using it as part of a larger expression. On it's own there's not really a lot of benefit to using it.